configs/wholebody/2d_kpt_sview_rgb_img/topdown_heatmap/coco-wholebody/hrnet_w48_coco_wholebody_384x288_dark_plus.py

```diff
@@ -1,5 +1,5 @@
 log_level = 'INFO'
-load_from = None
+load_from = 'https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w48_coco_384x288_dark-741844ba_20200812.pth'  # noqa: E501
```

Comment:
Why this is needed? Will the "pretrained" do the job?

Reply:
We want to load the weights of the head.

Reply:
Is it possible to support pretrained for heads?

Reply:
when `load_from` is not None, we should set `pretrained=None`.